app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -6,47 +6,42 @@ class MonthlyKeyMetricsReport < BaseReport
 
     attr_reader :report_date
 
+    Report = Struct.new(:metadata, :table, :csv_name)
+
     def perform(date = Time.zone.today)
       @report_date = date
 
-      account_reuse_table = account_reuse_report.account_reuse_report
-      total_profiles_table = account_reuse_report.total_identities_report
-      account_deletion_rate_table = account_deletion_rate_report.account_deletion_report
-      total_user_count_table = total_user_count_report.total_user_count_report
-
-      upload_to_s3(account_reuse_table, report_name: 'account_reuse')
-      upload_to_s3(total_profiles_table, report_name: 'total_profiles')
-      upload_to_s3(account_deletion_rate_table, report_name: 'account_deletion_rate')
-      upload_to_s3(total_user_count_table, report_name: 'total_user_count')
+      reports = []
+      reports << Report.new(
```

Comment:
WDYT of just building the reports array all in one statement instead of the `<<` ?
```suggestion
      reports = [
        Report.new(
```
etc